client/translations/en.json

```diff
@@ -3,8 +3,8 @@
   "search": "Search...",
   "language.fr": "🇫🇷",
   "language.en": "🇬🇧",
```

Comment:
add `header.` before them (search, language.fr and language.en) too and it's perfect 👌🏻